lib/voxpupuli/acceptance/serverspec_extensions/curl_command.rb

```diff
@@ -0,0 +1,56 @@
+# written by https://github.com/ekohl
+# https://github.com/mizzy/serverspec/pull/611 was rejected so adding it here.
+
+require 'serverspec'
+
+module Voxpupuli
+  module Acceptance
+    module ServerspecExtensions
+      class CurlCommand < Serverspec::Type::Command
+        def response_code
+          m = %r{Response-Code: (?<code>\d+)}.match(stderr)
+          return 0 unless m
+
+          m[:code].to_i
+        end
+
+        def body
+          command_result.stdout
+        end
+
+        def body_as_json
+          MultiJson.load(body)
+        end
+
+        private
+
+        def curl_command
+          command = "curl --silent --write-out '%{stderr}Response-Code: %{response_code}\\n' '#{@name}'"
```

Comment:
From the old code:
> curl on EL8 is too old to support --write-out %stderr

Is this something we need to account for, given overall we still support EL8 in voxpupuli-acceptance?

Reply:
Do we? And even if, the one vox module where we'd overwrite it dropped that hack, and I would doubt we'd add new tests before dropping el8

Reply:
I agree with @evgeni here.

Reply:
I still like a comment that it breaks on older curl versions but perhaps this discussion here is sufficient historic note

Reply:
I can add a comment with the lower bound for the version, that's easy!

Reply:
Then let's compromise there